brand/identity/index.md

```diff
@@ -190,57 +190,57 @@ layout: app
 <h3 class="m-bottom-2">Clanky</h3>
 	<div class="row">
 		<div class="col-sm-4 col-md-2">
-			<img class="m-bottom-0 p-bottom-0" src="{{site.baseurl}}/assets/images/mascot/m-1-1.png" alt="Dreamhost mascot" />
-			<p class="p-2 bg-c-g100">Format: <a href="{{site.baseurl}}/assets/downloads/mascot/m-1-1.eps">EPS</a></p>
+			<img class="m-bottom-0 p-bottom-0" src="{{site.baseurl}}/assets/images/mascot/new/m-1-1.png" alt="Dreamhost mascot" />
+			<p class="p-2 bg-c-g100">Format: <a href="{{site.baseurl}}/assets/images/mascot/new/m-1-1.png" download>PNG</a> | <a href="{{site.baseurl}}/assets/downloads/mascot/new/m-1-1.eps" download>EPS</a></p>
```

Comment:
Is there a reason for having `download` as an attribute in the HTML tag here?

Reply:
When I remove the attribute, it does show the image in a new tab. Should we have `download` attribute only for EPS?

Reply:
I think the images were always "right-click" > "Save Image" but not many people knew. I assume people like having an obvious "download link"

Reply:
Oh wait sorry this is for Identity page